bazaarvoice/emodb
# PD-195953: the SonarQube integration removing

## Github Issue #

[PD-1959531](https://bazaarvoice.atlassian.net/browse/PD-1959531)

## What Are We Doing Here?

Remove the SonarQube integration

## How to Test and Verify

1. Check out this PR
2. Run Command: `mvn clean install -DskipDependancyCheck=true -DskipTests=true`

## Risk

### Level 

`Low` 

### Required Testing

`Smoke`

### Risk Summary

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
